timemachine/cpp/src/kernels/k_nonbonded_deterministic.cuh

```diff
@@ -275,21 +345,23 @@ void __global__ k_nonbonded_inference(
     RealType sig_j = atom_j_idx < N ? params[lj_param_idx_sig_j] : 1;
     RealType eps_j = atom_j_idx < N ? params[lj_param_idx_eps_j] : 0;
 
+    RealType inv_cutoff = 1/cutoff;
+
     // In inference mode, we don't care about gradients with respect to parameters.
     for(int round = 0; round < 32; round++) {
 
```

Comment:
32 looks magic, perhaps use a named constant?

Reply:
32 is WARPSIZE - we will have a lot of problems on our hands if this is ever modified to be not 32